sky/utils/command_runner.py

```diff
@@ -68,6 +68,8 @@ def ssh_options_list(ssh_private_key: Optional[str],
         'ConnectTimeout': f'{timeout}s',
         # Agent forwarding for git.
         'ForwardAgent': 'yes',
+        # Disable loggings for ssh.
+        'LogLevel': 'QUIET',
```

Comment:
I think this change will also suppress helpful messages for users to debug their network.

E.g., I tried sky logs with wifi off:
```
# On master, wifi off
(base) romilb@romilbx1yoga:/mnt/d/Romil/Berkeley/Research/sky-experiments/examples$ sky logs myclus 2
Tailing logs of job 2 on cluster 'myclus'...
ssh: connect to host 3.81.184.201 port 22: Connection refused

# This branch, wifi off
(base) romilb@romilbx1yoga:/mnt/d/Romil/Berkeley/Research/sky-experiments/examples$ sky logs myclus 2
Tailing logs of job 2 on cluster 'myclus'...
(base) romilb@romilbx1yoga:/mnt/d/Romil/Berkeley/Research/sky-experiments/examples$  # no error
```

Reply:
Ahhh, that is a great catch! It was just for removing the `connection closed` logging, but it does not worth it to suppress the errors. Reverted. Thanks!